gapic-generator-cloud/templates/cloud/client/method/_init.erb

```diff
@@ -1,6 +1,5 @@
 <%- assert_locals method -%>
-<%= method.ivar %> = Google::Gax.create_api_call(
+<%= method.ivar %> = Google::Gax::ApiCall.new(
   @<%= method.service.stub_name %>.method(:<%= method.name %>),
```

Comment:
nit: still worth breaking the line now that there's only a couple params?

Reply:
Lol! this template is no longer being used, and should have been deleted. Removed.